process faults {6: ({1, 4}, 137957458)}
network faults {}
{10: ("62975AFD05C35CC0EC578D2F5FD7362C397666F2366D3AAF6223876B3FB284F3", {4, 0, 6, 5, 1, 2, 3}), 8: ("0F4EEC767065233BA15CA59BA515C8DC0680227746C6B99EBE7C40C85CFF1515", {6, 2, 1, 5, 4, 3, 0}), 7: ("593EC6EA6D1664E2A55ACA052397F9020D6B14E4E1208423B5E38EE975651A7D", {2, 0, 3, 4, 6, 1, 5}), 5: ("FB97C7D273031A8D70CABEAB68CE596F73FAFDB87754BA3FD25400CA17F7462B", {4, 3, 6, 5, 0, 1, 2}), 9: ("ED137A2C8CD97BD55096ED45DF2FD69102E4F7BB9A3A11693313405FB6985BEA", {4, 2, 1, 6, 0, 5, 3}), 6: ("3CFB19E5805C777A42B961C40C22917EF7AD45CF49C280E8EC69DC8B495EC9EE", {1, 0, 4, 2, 3, 5, 6})}
7
reason: all committed
done!
